lib/faye_publisher.rb

```diff
@@ -182,8 +182,8 @@ def dialog_channels message, channels
     channels + ["/dialog-#{[message.from_id, message.to_id].sort.join '-'}"]
   end
 
-  def forum_channel forum_id, locale
-    "/forum-#{forum_id}/#{locale}"
+  def forum_channel forum_id
+    "/forum-#{forum_id}/ru"
```

Comment:
@Yokitori см пред коммент
```suggestion
    "/forum-#{forum_id}"
```